content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml

```diff
@@ -86,12 +86,12 @@
                                         text="Display caption as pop-up"
                                         uncheckedValue="false"
                                         value="{Boolean}true"/>
-                                    <disableUuid
+                                    <disableUuidTracking
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                        fieldDescription="Check to disable UUID of the asset."
+                                        fieldDescription="Check to disable tracking of assset's UUID."
```

Comment:
@bpauli - we introduced a spelling mistake here from Vlad's comment, too many "s"